src/Details/Details.js

```diff
@@ -1,51 +1,78 @@
 import './Details.css';
+import { NavLink, useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react'
 
-const Details = ({ data }) => {
+const Details = ({ allParks }) => {
+
+  // const [park, setPark] = useState(null)
+
+  const parkCode = useParams().parkCode
+
+  const park = allParks.data.find(park => park.parkCode === parkCode)
+
+  // useEffect(() => {
+  //   fetch(`https://developer.nps.gov/api/v1/parks?parkCode=${parkCode}&limit=1&start=0&api_key=l6jn2TRgOT3bXFR8Fk7iAF7OP6Bkf7lslJE9TMMX`)
+  //     .then((res) => res.json()) 
+  //     .then((res) => {
+  //       setPark(res.data[0])
+  //     })
+  //     .catch((error) => {
+  //       console.error('Error fetching data:', error);
+  //       setPark(null); 
+  //     });
+  // }, [park])
```

Comment:
I'm thinking you should just uncomment this `useEffect` and `useState` since it works and is able to refresh and go back and forth.  A single extra fetch call isn't gonna add too much strain.

Reply:
I think why it's breaking on refresh is because there's no state set